src/estimators/bloom_filters.py

```diff
@@ -545,3 +546,143 @@ def _denoise_one(self, sketch):
     denoised_sketch.sketch = 1 - expected_zeros / (
         1 - self._probability[1] - self._probability[0])
     return denoised_sketch
+
+
+class SketchOperator(object):
+  """Sketch operations for supporting local DP frequency dedupe."""
+
+  BAYESIAN_APPROXIMATION = "bayesian"
```

Comment:
Per offline discussion. Move sketch operators to "bloom_filter_sketch_operators.py" and create a base class SketchOperator with two derived class, one each for bayesian and expectation.

Reply:
Updated. Please review!